packages/create-toolpad-app/tests/index.spec.ts

```diff
@@ -35,8 +35,9 @@ async function waitForMatch(input: Readable, regex: RegExp): Promise<RegExpExecA
   });
 }
 
-test(
-  'create-toolpad-app can bootstrap a Toolpad app',
+// @TODO: Unskip once @mui/toolpad-studio is released for the first time.
```

Comment:
We can unskip this after we publish, the test fails because `@mui/toolpad-studio` hasn't been published yet.